client/my-sites/plan-features-2023-grid/features.tsx

```diff
@@ -1,10 +1,41 @@
-import { getPlanClass, FEATURE_CUSTOM_DOMAIN } from '@automattic/calypso-products';
+import { getPlanClass, FEATURE_CUSTOM_DOMAIN, isFreePlan } from '@automattic/calypso-products';
+import styled from '@emotion/styled';
 import classNames from 'classnames';
 import { localize } from 'i18n-calypso';
+import { useGetWordPressSubdomain } from './hooks/use-get-wordpress-subdomain';
 import { PlanFeaturesItem } from './item';
 import { Plans2023Tooltip } from './plans-2023-tooltip';
 import { TransformedFeatureObject } from './types';
 
+const SubdomainSuggestion = styled.div`
+	.is-domain-name {
+		position: absolute;
+		top: -15px;
+		color: var( --studio-gray-50 );
+		text-decoration: line-through;
+	}
+`;
+
+const LoadingPlaceHolder = styled.div`
+	margin: 0;
+`;
+
+const FreePlanCustomDomainFeature: React.FC< { domainName: string } > = ( { domainName } ) => {
+	const {
+		data: wordPressSubdomainSuggestion,
+		isLoading,
+		isError,
+	} = useGetWordPressSubdomain( domainName );
+
+	return (
+		<SubdomainSuggestion>
+			<div className="is-domain-name">{ domainName }</div>
+			{ isLoading && <LoadingPlaceHolder className="async-load__placeholder" /> }
```

Comment:
Non-blocker: it's intriguing to see that this "loading placeholder" has been used this way across calypso, e.g. https://github.com/Automattic/wp-calypso/blob/trunk/Users/southp/a8c/wp-calypso/client/components/async-load/index.jsx#L14 seems to be the origin.

Should we consider to extract it into a component, say `<AsyncLoadPlaceholder />` under `/component/async-load/`?

Reply:
That's a good point, the app has many different loading components, but it is challenging to choose the correct one without documentation. On second thoughts, I don't think we should use this one since it relies on the class name being present, which is not guaranteed. I'll replace this with something else.